.github/workflows/publish-release.yml

```diff
@@ -91,17 +89,12 @@ jobs:
           node-version: lts/*
       - name: install
         run: npm ci
-      - name: sync versions
-        run: npm run sync:version
-      - name: get version
-        id: build_versions
-        run: node tools/scripts/get-build-versions.js
       - name: re-publish internal loader (prod)
         run: |
           node ./tools/scripts/upload-ab-to-s3.js \
             --env prod \
             --current https://js-agent.newrelic.com/nr-loader-spa-current.min.js \
-            --next https://js-agent.newrelic.com/nr-loader-spa-${{steps.build_versions.outputs.CDN_VERSION}}.min.js \
+            --next https://js-agent.newrelic.com/nr-loader-spa-$(cat package.json | jq -r '.version').min.js \
```

Comment:
Have you verified the image used by the job container contains these binaries? I just pulled `ubuntu:latest` and it does not have `jq` installed by default. You may want to add it to the `Setup Container` step in all the jobs where it is needed.

Reply:
Good catch. Thanks!